modules/cache/jvm/src/main/scala/coursier/cache/FileCache.scala

```diff
@@ -31,10 +31,12 @@ import scala.util.control.NonFatal
   localArtifactsShouldBeCached: Boolean = false,
   followHttpToHttpsRedirections: Boolean = true,
   followHttpsToHttpRedirections: Boolean = false,
+  rejectNonAuthoritativeResponses: Boolean = false,
   maxRedirections: Option[Int] = CacheDefaults.maxRedirections,
   sslRetry: Int = CacheDefaults.sslRetryCount,
   sslSocketFactoryOpt: Option[SSLSocketFactory] = None,
   hostnameVerifierOpt: Option[HostnameVerifier] = None,
+  authRealmOpt: Option[String] = None,
```

Comment:
Just like for `ConnectionBuilder` above, these new fields should be added in last position, and after a `@since("2.0.17")` annotation, not to break binary compatibility.